src/hooks/useDeskproTag.ts

```diff
@@ -0,0 +1,60 @@
+import { useCallback } from "react";
```

Comment:
This file is kinda large and hard to review, can it be broken down?

Reply:
I’m afraid I’ve made it as small as possible. If I try to shorten it further, readability will suffer.